pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -50,6 +50,13 @@ message ApplicationConfiguration {
 
   // Common configuration options that apply to all Buildbarn binaries.
   buildbarn.configuration.global.Configuration global = 19;
+
+  // Install devices into /dev of the input root. Useful in combination
+  // with having the runner chroot into the input root.
+  bool install_devices = 20;
```

Comment:
Do we need the additional boolean flag? Doesn’t a non-empty devices list already imply the boolean value?